compiler/control/OMROptions.cpp

```diff
@@ -629,7 +629,7 @@ TR::OptionTable OMR::Options::_jitOptions[] = {
    {"disableZEC12",                       "O\tdisable zEC12 support",                          SET_OPTION_BIT(TR_DisableZEC12), "F" },
    {"disableZHelix",                      "O\t[Deprecated] alias for disableZEC12",                          SET_OPTION_BIT(TR_DisableZEC12), "F"},
    {"disableZImplicitNullChecks",         "O\tdisable implicit null checks on 390",            SET_OPTION_BIT(TR_DisableZImplicitNullChecks), "F"},
-   {"disableZNext",                       "O\tdisable zNext support",                        SET_OPTION_BIT(TR_DisableZNext), "F"},
+   {"disablez17",                         "O\tdisable z17 support",                            SET_OPTION_BIT(TR_Disablez17), "F"},
```

Comment:
Please use capitalized Z
```suggestion
   {"disableZ17",                         "O\tdisable z17 support",                            SET_OPTION_BIT(TR_DisableZ17), "F"},
```
Same suggestion for capitalizing it for enable option. 

Also I think this will fail functionally when you supply either option. Options in this array are sorted so `disableZ17` should come after  `disableZ16` and `enableZ17` should come after `enableZ16`.